packages/nitro-protocol/src/contract/multi-asset-holder.ts

```diff
@@ -1,15 +1,15 @@
-import {utils, BigNumber, ethers} from 'ethers';
+import {utils, BigNumber, ethers, constants} from 'ethers';
+import ExitFormat from '@statechannels/exit-format';
```

Comment:
Is this a long-term decision?

Reply:
I would say it is a medium term decision. 

It shouldn't be difficult to absorb that package into `nitro-protocol` if we want to in the future. Keeping it separate makes it easier for other projects to use.